main.py

```diff
@@ -128,12 +135,7 @@ def main():
 
 
 def generate_travel_route(
-    travel_df,
-    geocodes_df,
-    start_lat,
-    start_lon,
-    distance_to_start,
-    total_distance,
+    travel_df, geocodes_df, start_lat, start_lon, distance_to_start, total_distance,
 ):
```

Comment:
It should be possible to calculate total distance just from travel_df.
Also, maybe start_lat, start_lon can be also found in travel_df?